libr/core/linux_heap_glibc.c

```diff
@@ -21,6 +21,39 @@
 #define GHT_MAX UT64_MAX
 #endif
 
+
+static GHT GH(get_va_symbol)(RCore *core, const char *path, const char *symname) {
+	GHT vaddr = GHT_MAX;
+	RListIter *iter;
+	RBinSymbol *s;
+	// RBin *bin = core->bin;
+	RBin *bin = r_bin_new();
```

Comment:
dont create new instances of rbin, just use core->rbin

Reply:
Yeah I was just trying to see if with r_bin_new the tests worked, as I did in a1dca99 is fine?